app/services/monitor_index_jobs.rb

```diff
@@ -26,10 +30,17 @@ def notify_sentry_for_dead_letter_queue
       dead_job = @dead_queue.read
       break if dead_job.nil?
 
-      Rails.logger.info("MonitorIndexJobs: Sending job #{dead_job.class.to_s} to dead letter queue")
-      Raven.capture_message("Job Found in Dead Letter Queue", :extra => dead_job.inspect)
+      begin
+        log_info("Sending job #{dead_job.class.to_s} to dead letter queue")
+        Raven.capture_message("Job Found in Dead Letter Queue", :extra => dead_job.inspect)
+
+        dead_job.cleanup_after_call   #delete the message so we dont keep sending to Sentry
 
-      dead_job.cleanup_after_call   #delete the message so we dont keep sending to Sentry
+        @processed_from_dead += 1
+      rescue => ex
+        Raven.capture_exception(ex)
+        log_error("dead job error #{ex.message} process #{dead_job.inspect}")
```

Comment:
somehow I'd never encountered `log_error` and `_info` - TIL they're part of action_controller which I'm guessing is mixed in to here?

Reply:
This is from the `prefix_logger.rb` in this PR